rules/rules-to-take-effective-notes/rule.md

```diff
@@ -0,0 +1,42 @@
+---
+type: rule
+title: Do you know how to take effective notes?
+uri: rules-to-take-effective-notes
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+created: 2023-03-30T23:48:50.319Z
+guid: d96efe9b-8981-44d0-b22e-b41a7d173db8
+---
+Taking effective notes is an important skill that can help you stay organized and remember important information. Whether you’re in a meeting with a client or jotting down tasks for your to-do list, it’s important to have a system in place for taking and organizing your notes.
+
+One key tip for taking effective notes is to avoid deleting old notes. It may be tempting to delete notes once they’re no longer relevant, but it’s often better to archive them or move them to a different location where they can still be accessed if needed. This way, you won’t regret deleting valuable information later on. However, moving it to a different place may make it hard to find.
+
+An even better way to take notes is to include metadata such as the date and time when the note was taken, as well as your name. This can help you keep track of when and by whom the note was taken, making it easier to find and reference specific notes later on.
+
+Here are a few examples of how this can be done:
+
+::: greybox
+
+**e.g. #1**. On an Invoice page with a notes field
+
+*    21/2/2023 Ravi - Spoke to Bob and they said we are. 
+*    19/2/2023 Steven Q. - Spoke to Alex and they said we suck
+  :::

```

Comment:
Thinking about it more, I think this Greybox would be clearer as a screenshot of a timepro invoice that has these notes in it